gprofiler/main.py

```diff
@@ -572,10 +573,19 @@ def parse_cmd_args() -> configargparse.Namespace:
         " as well",
     )
 
+    parser.add_argument(
+        "--collect-databricks-job-name",
+        action="store_true",
+        dest="collect_databricks_job_name",
+        default=False,
+        help="gProfiler will collect Databricks job name for ephemeral cluster",
+    )
+
     args = parser.parse_args()
 
     args.perf_inject = args.nodejs_mode == "perf"
-
+    args.pid_ns_check = False if args.collect_databricks_job_name else args.pid_ns_check
```

Comment:
Let's do this change in another PR, not this one (changing `pid_ns_check` and `perf_mode`)